core-services/image-mirroring/openshift/mapping_origin_4_10

```diff
@@ -92,6 +92,7 @@ registry.ci.openshift.org/origin/4.10:csi-driver-manila quay.io/openshift/origin
 registry.ci.openshift.org/origin/4.10:csi-driver-manila-operator quay.io/openshift/origin-csi-driver-manila-operator:4.10 quay.io/openshift/origin-csi-driver-manila-operator:4.10.0 quay.io/openshift/origin-csi-driver-manila-operator:latest
 registry.ci.openshift.org/origin/4.10:csi-driver-nfs quay.io/openshift/origin-csi-driver-nfs:4.10 quay.io/openshift/origin-csi-driver-nfs:4.10.0 quay.io/openshift/origin-csi-driver-nfs:latest
 registry.ci.openshift.org/origin/4.10:csi-driver-shared-resource quay.io/openshift/origin-csi-driver-shared-resource:4.10 quay.io/openshift/origin-csi-driver-shared-resource:4.10.0 quay.io/openshift/origin-csi-driver-shared-resource:latest
+registry.ci.openshift.org/origin/4.10:csi-driver-shared-resource-mustgather quay.io/openshift/origin-csi-driver-shared-resource-mustgather:4.10 quay.io/openshift/origin-csi-driver-shared-resource-mustgather:4.10.0 quay.io/openshift/origin-csi-driver-shared-resource-mustgather:latest
```

Comment:
name looks ok to me, I debated if it should be `must-gather` instead, but we arleady have `origin-local-storage-mustgather` (along with `origin-must-gather`) so it seems like the decision is made.

cc @smarterclayton to confirm my naming approval